src/private/files.ts

```diff
@@ -416,4 +453,21 @@ export namespace files {
       callback,
     );
   }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Gets list of downloads for current user
+   * @param callback Callback that will be triggered post downloads load
+   */
+  export function getFileDownloads(callback: (error: SdkError, files: IFileItem[]) => void): void {
```

Comment:
The callback should have both error and files as optional, to better represent the data that'll actually be provided back when it's invoked.

Reply:
Comment incorporated.